src/hotspot/os/posix/perfMemory_posix.cpp

```diff
@@ -135,38 +136,39 @@ static void save_memory_to_file(char* addr, size_t size) {
 
 // Shared Memory Implementation Details
 
-// Note: the solaris and linux shared memory implementation uses the mmap
+// Note: the linux shared memory implementation uses the mmap
```

Comment:
Is this really specific to Linux?

Reply:
It is not Linux specific.  AIX and BSD also use mmap.  So I changed the comment to say "Note: the Posix shared ..."